src/components/Identification/index.js

```diff
@@ -85,48 +87,117 @@ export const Identification = props => {
+            : socialMediaInfo.facebook.prefix + props.profile.Facebook,
         );
         setTwitterLink(
-          !profile.Twitter || profile.Twitter === ''
+          !props.profile.Twitter || props.profile.Twitter === ''
             ? ''
-            : socialMediaInfo.twitter.prefix + profile.Twitter,
+            : socialMediaInfo.twitter.prefix + props.profile.Twitter,
         );
         setLinkedInLink(
-          !profile.LinkedIn || profile.LinkedIn === ''
+          !props.profile.LinkedIn || props.profile.LinkedIn === ''
             ? ''
-            : socialMediaInfo.linkedIn.prefix + profile.LinkedIn,
+            : socialMediaInfo.linkedIn.prefix + props.profile.LinkedIn,
         );
         setInstagramLink(
-          !profile.Instagram || profile.Instagram === ''
+          !props.profile.Instagram || props.profile.Instagram === ''
             ? ''
-            : socialMediaInfo.instagram.prefix + profile.Instagram,
+            : socialMediaInfo.instagram.prefix + props.profile.Instagram,
         );
       } catch (error) {
         // Do Nothing
       }
     }
 
     loadUserProfile();
-  }, []);
+  }, [props.profile, props.myProf]);
+
+  /**
+   * Gets the current breakpoint according to Material-UI's breakpoints
+   */
+  useEffect(() => {
+    if (props.myProf) {
+      // Gets the current Material-UI Breakpoint
+      function getMaterialUIBreakpoint(width) {
+        let currentWidth = '';
+        // If current width is in Material-UI breakpoint XS
+        if (width >= windowBreakWidths.breakXS && width < windowBreakWidths.breakSM) {
+          currentWidth = 'xs';
+        }
+        // If current width is in Material-UI breakpoint SM
+        else if (width >= windowBreakWidths.breakSM && width < windowBreakWidths.breakMD) {
+          currentWidth = 'sm';
+        }
+        // If current width is in Material-UI breakpoint MD
+        else if (width >= windowBreakWidths.breakMD && width < windowBreakWidths.breakLG) {
+          currentWidth = 'md';
+        }
+        // If current width is in Material-UI breakpoint LG
+        else if (width >= windowBreakWidths.breakLG && width < windowBreakWidths.breakXL) {
+          currentWidth = 'lg';
+        }
+        // If current width is in Material-UI breakpoint XL
+        else if (width >= windowBreakWidths.breakXL) {
+          currentWidth = 'xl';
+        }
+        return currentWidth;
+      }
+
+      // An event listener for when the browser size changes to get the current Material-UI breakpoint
+      window.addEventListener('resize', event => {
+        setCurrentWidth(getMaterialUIBreakpoint(event.target.innerWidth));
+      });
+      // Sets the current Material-UI Breakpoint
+      setCurrentWidth(getMaterialUIBreakpoint(window.innerWidth));
+      // Removes the resize window event listener
+      return window.removeEventListener('resize', () => {});
+    }
+  }, [props.myProf]);
```

Comment:
Added function that determines the window's current Material-UI's current breakpoint